.github/workflows/ci.yml

```
@@ -202,15 +202,26 @@ jobs:
     name: 👘 ・Synpress
     runs-on: ubuntu-20.04
     container: cypress/browsers:node16.5.0-chrome97-ff96
-    if: github.actor != 'dependabot[bot]'
+    if: ${{ github.event.pull_request.merged || (github.event.pull_request.assignee.login == 'niemam29') }}
     env:
-      TEST_PARAMS: "-s 'tests/synpress/specs/${{ matrix.containers }}/*/*.ts'"
+      IF_MERGED: ${{ github.event.pull_request.merged }}
       PRIVATE_KEY: ${{ secrets.TEST_WALLET_PRIVATE_KEY }}
+      ASSIGNEE: ${{ github.event.pull_request.assignee.login }}
     strategy:
       fail-fast: false
       matrix:
         containers: [02transactionfull, 01transactionless]
     steps:
+      - name: Check how tests should be executed #1
+        if: ${{ (github.event.pull_request.assignee.login == 'niemam29') }}
```

```suggestion
        if: ${{ (github.event.pull_request.assignee.login == 'niemam29') }}
```

Are you sure you need to hardcode your id here?

---

I was looking for a solution to run reporter on demand, but couldnt get access to commit message from the level of pr. I will try one more thing that popped into my head, but if you got some other idea that would be better than this please let me know